k8s/gpu-training.yaml

```diff
@@ -1,28 +1,25 @@
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: "gpu-training-deployment"
+  name: "gpu-training"
 spec:
   replicas: 1
+  selector:
+    matchLabels:
+      app: "gpu-training"
   template:
     metadata:
       labels:
         app: "gpu-training"
     spec:
       containers:
       - name: "gpu-training"
-        image: ${REPO}/almond/training:v0.7
+        image: stanfordoval/almond-cloud:latest-decanlp-cuda
         imagePullPolicy: Always
-        env:
-        - name: "PORT"
-          value: "9999"
-        - name: "RUN_GPU_DAEMON"
-          value: "true"
+        command: ["run-gpu-training"]
```

Comment:
should this be args instead?
https://kubernetes.io/docs/tasks/inject-data-application/define-command-argument-container/

Reply:
Yes it should, thanks for spotting it.